api/envoy/extensions/access_loggers/open_telemetry/v3/logs_service.proto

```diff
@@ -46,4 +46,9 @@ message OpenTelemetryAccessLogConfig {
   // See 'attributes' in the LogResource proto for more details.
   // Example: ``attributes { values { key: "user_agent" value { string_value: "%REQ(USER-AGENT)%" } } }``.
   opentelemetry.proto.common.v1.KeyValueList attributes = 3;
+
+// Optional additional prefix to use when emitting statistics. This allows to distinguish
+  // emitted statistics between configured open telemetry access loggers in the logger chain.
+  // If empty, the default `access_logs.open_telemetry_access_log.` will be used.
+  string stat_prefix = 6;
```

Comment:
Please add a PGV constraint to ensure that if this is populated, it is not empty:
`[(validate.rules).string = {min_len: 1}]`

Reply:
Unnecessary for optional string field?